dataeng/jobs/createJobs.groovy

```diff
@@ -25,13 +27,15 @@ try {
 }
 
 def taskMap = [
-    ANALYTICS_EMAIL_OPTIN_JOB: AnalyticsEmailOptinJob,
-    ANALYTICS_EXPORTER_JOB: AnalyticsExporterJob,
-    USER_ACTIVITY_JOB: UserActivityJob,
-    VIDEO_TIMELINE_JOB: VideoTimelineJob,
-    USER_LOCATION_BY_COURSE_JOB: UserLocationByCourseJob,
-    ENROLLMENT_JOB: EnrollmentJob,
-    MODULE_ENGAGEMENT_JOB: ModuleEngagementJob,
+//    ANALYTICS_EMAIL_OPTIN_JOB: AnalyticsEmailOptinJob,
+//    ANALYTICS_EXPORTER_JOB: AnalyticsExporterJob,
+//    USER_ACTIVITY_JOB: UserActivityJob,
+//    VIDEO_TIMELINE_JOB: VideoTimelineJob,
+//    USER_LOCATION_BY_COURSE_JOB: UserLocationByCourseJob,
+//    ENROLLMENT_JOB: EnrollmentJob,
+//    MODULE_ENGAGEMENT_JOB: ModuleEngagementJob,
```

Comment:
Oh, interesting.  You have to comment these out so they don't get overwritten?   Or so they don't fail?

Reply:
This is part of a second commit that i would delete before merging.  Our job seeding infrastructure updates all jobs at once, but for testing I want to limit the blast radius.  Perhaps we should revisit how aggressive the createJobs job is.